src/pages/index.js

```diff
@@ -59,13 +71,28 @@ const cardModalCloseBtn = cardModal.querySelector(".modal__close-btn");
 const cardNameInput = cardModal.querySelector("#add-card-name-input");
 const cardLinkInput = cardModal.querySelector("#add-card-link-input");
 
+// Avatar form elements
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarForm = avatarModal.querySelector(".modal__form");
+const avatarSubmitBtn = avatarModal.querySelector(".modal__submit-btn");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+
+// Delete form elements
+const deleteModal = document.querySelector("#delete-modal");
+const deleteForm = deleteModal.querySelector(".delete__form");
+const cancelButton = deleteModal.querySelector(".modal__cancel-btn");
+const closeButton = deleteModal.querySelector(".modal__close-btn");
```

Comment:
`closeButton `

There are 3 modals with different buttons and forms in the project. You need to **name** variables and functions the way all could understand what they are: `profileForm`, `profileCloseButton` and so on


`deleteModalCloseButton`